usvm-core/src/main/kotlin/org/usvm/api/collection/ObjectMapCollectionApi.kt

```diff
@@ -61,7 +62,9 @@ object ObjectMapCollectionApi {
                 with(ctx) {
                     val boundConstraint = mkSizeGeExpr(length, mkSizeExpr(0))
                     // Map size must be correct regardless of guard
-                    assert(boundConstraint) ?: return null
+                    assert(boundConstraint)
+                        .logAssertFailure { "SymbolicMap size correctness constraint" }
```

Comment:
Let's add a common word for all these messages to be able to find them in logs files. Something like "Constaint violation: Symbolic..."